src/api/directives.ts

```diff
@@ -74,4 +79,27 @@ const submitDirective = async (
     }
 };
 
-export { exchangeBearerToken, submitDirective };
+const getApplieDirectives = (type: keyof typeof DIRECTIVES, userId: string) => {
```

Comment:
>`getApplieDirectives`

There is a typo in this function name. The _d_ in _Applied_ is missing (i.e., `getAppliedDirectives`).

Reply:
Well that explains why I kept having issues finding this function when searching for it!

Fixed